var/spack/repos/builtin/packages/doxygen/package.py

```diff
@@ -18,6 +18,8 @@ class Doxygen(CMakePackage):
     homepage = "https://www.doxygen.org"
     url = "https://github.com/doxygen/doxygen/archive/refs/tags/Release_1_9_5.tar.gz"
 
+    version("1.9.8", sha256="691777992a7240ed1f822a5c2ff2c4273b57c1cf9fc143553d87f91a0c5970ee")
```

Comment:
How did you calculate this?  

I'm getting the following from "spack checksum doxygen 1.9.8" instead:
`77371e8a58d22d5e03c52729844d1043e9cbf8d0005ec5112ffa4c8f509ddde8`

Reply:
Yes.  Stupid cut&paste error.  I have your has in my editor but copied the wrong one.  It should be fixed in [f7bd57c](https://github.com/spack/spack/pull/41123/commits/f7bd57cdb9f54eacd26e1e4aad309154ba01c6a5).